### DISCUSSION TITLE
constant errors with notify call service in 0.48.0

### DISCUSSION THREAD

**Discussion by ktoms71** [zachowj/node-red-contrib-home-assistant-websocket#838] [Q&A]
Hi All,   I have a head scratcher of sorts.   Im running HA and node red separate docker containers.   I can use the call service node to perform functions like turning on lights.   But Im receiving errors when using the notify feature.   In NR, Im using node-red-contrib-home-assistant-websocket version 0.48.0    My message object is well formatted: {
    "message": "The garage door has been open for 10 minutes.",
    "title": "Your Garage Door Friend",
    "target": "platform specific",
    "data": "platform specific"
} 
But Im consistently receiving an api error  "Call-service error. expected dict for dictionary value @ data['data']"   Any thoughts?

as info, Im on HA core 2022.12.8 and this web socket palette 0.48.0.   I tested the notify service to my cell phone from dev tools tab in HA and it functions properly.

**Comment by sym0nd0** [comment]
Have you tried:

```
{
    "title": "Your Garage Door Friend", 
    "message": "The garage door has been open for 10 minutes."
} 
```
Double check the formatting, I'm on mobile atm.

**Comment by mike5879** [comment]
I am having the same error trying to send notifications to the mobile app using the example message.  Any help would be appreciated since I am moving most of my automations from HA to nodered
